src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -56,11 +56,11 @@
      * Returns an iterable view of all {@link Slot}s (leaf nodes) in this
      * Inventory.
      *
-     * @param <T> expected inventory type, specified as generic to allow easy
+     * @param <T> expected slot type, specified as generic to allow easy
      *      pseudo-duck-typing
      * @return an iterable view of all Slots (leaf nodes) in this inventory
      */
-    <T extends Inventory> Iterable<T> slots();
+    <T extends Slot> Iterable<T> slots();
```

Comment:
K, I'll remove this one. I think @ST-DDT pinged you about it in the OCD list, I just yolo'd it in.